src/Command/Argument/Search/SchemaFields/TagField.php

```diff
@@ -21,16 +21,19 @@ class TagField extends AbstractField
      * @param bool        $noIndex
      * @param string      $separator
      * @param bool        $caseSensitive
+     * @param bool        $allowsEmpty
      */
     public function __construct(
         string $identifier,
         string $alias = '',
         $sortable = self::NOT_SORTABLE,
         bool $noIndex = false,
         string $separator = ',',
-        bool $caseSensitive = false
+        bool $caseSensitive = false,
+        bool $allowsEmpty = false,
```

Comment:
Should `$allowEmpty` be the last one, to match the other arguments from the parent? Might be too late for that, given that other parameters sneaked in already after `$noIndex`.

Reply:
Maybe I didn't get it, `$allowsMissing` is the last one across all field